### DISCUSSION TITLE
PR 的提交，必须按照顺序，对吧？

### DISCUSSION THREAD

**Discussion by linghuccc** [openbuildxyz/Web3-Frontend-Bootcamp#537] [Q&A] [answered]
虽然提交规范里面提到了，“`实际上是可以开启多个 PR 的，可「多线程」去完成任务`”，但在真正提交 PR 的时候，并不能多线程提交，只能按照顺序，一个任务一个任务提交，对吧？

只有提交了 init 之后，才能提交 task1；只有提交了 task1 之后，才能提交 task2； 。。。

是这样吧？

虽然 PR 规范里面提到，“`请不要把报名和提交TASK的内容合并到一起，也不要将多个TASK放到一起提交！`”

但实际上，task1 的内容必然包含报名内容；task2 的内容必然包含报名内容和 task1 的内容；。。。。

对吧？

**Comment by ourai** [comment, marked as answer]
显然，既然规范里那么写了，就是可实现的，我就以一个刚参加这个训练营的人的背景为例——

先把这个仓库 fork 到自己的空间，然后在电脑本地 `git clone` 自己空间中的仓库。

**这时默认是在 `main` 分支，你可以通过 CLI 或 GUI 基于当前的提交节点创建出 `init`、`task1`、`task2` 等多个分支，在对应的分支上完成相应的任务。**

**每个任务完成后推送到你自己空间的远程仓库，在创建 PR 时选择相应的分支合到本训练营仓库的 `main` 分支。**

每个任务分支上理论上并不会包含其他任务的内容。

仍有疑问的话，请看视频《[OpenBuild Web3 前端训练营作业提交演示](https://www.bilibili.com/video/BV11gVxeuE7Y/)》。

更多细节可查阅 Git 与 GitHub 官方文档，或者询问各类 GPT 工具。
